encrypted-media-respec.html

```diff
@@ -2493,13 +2493,25 @@
             <li><p>Let <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
             <li>
               <p>
-                If <var>session</var>'s <var>session type</var> is <a def-id="persistent-usage-record-session"></a>,
-                store <var>session</var>'s <var>record of key usage</var>, if it exists.
+                If <var>session</var>'s <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, execute the following steps:
```

Comment:
"...  in parallel"

This is the only remaining CDM use that is not in parallel. The rest of this algorithm is running (in a task) on the main event loop.